.github/container/levanter-cache-warn.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+YELLOW='\033[0;33m'
+NOCOLOR='\033[0m'
+
+echo -e "${YELLOW}
+WARNING: To speed up the model training it is recommended to cache the datasets
```

Comment:
The message should ask the user to make sure that `--cache_dir` is a **shared** filesystem across nodes.
In a HPC cluster environment, a common practice is to mount a workspace/project folder from a distributed file system into the designated container path.

Reply:
Looks like done